Formula/glib-openssl.rb

```diff
@@ -0,0 +1,58 @@
+class GlibOpenssl < Formula
+  desc "OpenSSL GIO module for glib"
+  homepage "https://launchpad.net/glib-networking"
+  url "https://download.gnome.org/sources/glib-openssl/2.50/glib-openssl-2.50.4.tar.xz"
+  sha256 "eda914e29d9d72f03185aba92c33ec3949862c05b73b8eb04d1e1b6217b756c0"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "openssl"
+
+  link_overwrite "lib/gio/modules"
+
+  def install
+    # Install files to `lib` instead of `HOMEBREW_PREFIX/lib`.
+    inreplace "configure", "$($PKG_CONFIG --variable giomoduledir gio-2.0)", lib/"gio/modules"
```

Comment:
there's no configure arg or envvar for this?

Reply:
nope

Reply:
ok